web/src/common/Header.tsx

```diff
@@ -107,6 +107,7 @@ export const Header = (): JSX.Element => {
         <Popover.Content>
           <VersionText />
           <p>Person of contact: Eleni Pantelli (elp@equinor.com)</p>
+          <p>Technical Support: team-hermes@equinor.com</p>
```

Comment:
Have this email been tested? I expected a test email, but I never got one

Reply:
updated and tested ok